.readthedocs.yaml

```diff
@@ -1,6 +1,9 @@
 # This is the version for the readthedocs configuration. Version 2 ignores
 # web-based configuration and uses everything from this file.
 version: 2
+sphinx:
+  fail_on_warning: true
+  configuration: docs/source/conf.py
```

Comment:
required for recent sphinx change

Reply:
Yep we did this on vivarium_research recently!

Reply:
@stevebachmeier You can delete this since you will need to rebase and that would probably make it easier for you